lib/node_modules/@stdlib/stats/base/dists/beta/mean/README.md

```diff
@@ -144,12 +144,102 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
-<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+<!-- C interface documentation. -->
 
-<section class="references">
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/beta/mean.h"
+```
+
+#### stdlib_base_dists_beta_mean( alpha, beta )
+
+Returns the [expected value][expected-value] of a [beta][beta-distribution] distribution with parameters `alpha` (first shape parameter) and `beta` (second shape parameter).
+
+```c
+double out = stdlib_base_dists_beta_mean( 1.0, 1.0 );
+// returns 0.5
+```
+
+The function accepts the following arguments:
+
+-   **alpha**: `[in] double` first shape parameter.
+-   **beta**: `[in] double` second shape parameter.
+
+```c
+double stdlib_base_dists_beta_mean( const double a, const double b );
+```
 
 </section>
 
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/beta/mean.h"
+#include "stdlib/constants/float64/eps.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+static double random_uniform( const double min, const double max ) {
+    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
+    return min + ( v*(max-min) ) + STDLIB_CONSTANT_FLOAT64_EPS;
+}
+
+int main( void ) {
+    double alpha;
+    double beta;
+    double y;
+    int i;
+    
+    for ( i = 0; i < 25; i++ ) {
+        alpha = random_uniform( 0.0, 10.0 );
+        beta = random_uniform( 0.0, 10.0 );
+        y = stdlib_base_dists_beta_mean( alpha, beta );
+        printf( "alpha: %lf, beta: %lf, F(X;a,b): %lf\n", alpha, beta, y );
+    }
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+</section>
+
+<!-- /.c -->
+
 <!-- /.references -->
```

Comment:
Let's keep the references section:

```suggestion
<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->

<section class="references">

</section>

<!-- /.references -->
```

Reply:
it was actually removed in arcsine cdf so i thought of removing it , now will add this in each PR